bitbucket/credentials.py

```diff
@@ -0,0 +1,42 @@
+from django.utils.functional import cached_property
+
+from .base import KatkaService
+from .conf import settings
+from .exceptions import katka_service_exception_to_api
+
+
+class CredentialsProvider:
+    @cached_property
+    def access_token(self):
+        raise NotImplementedError('This method should be implemented by the implementation classes')
+
+
+class KatkaCredentialsService(CredentialsProvider, KatkaService):
+    def __init__(self, request, credential_public_id: str = None):
+        self.credential_public_id = credential_public_id
+
+        self.base_url = settings.KATKA_SERVICE_LOCATION.rstrip('/')
+        self.base_path = 'credentials'
```

Comment:
Maybe store this path in a settings variable

Reply:
If I do that here I should also do that for secrets part and for bitbucket paths. I don't think that is needed, at least for now.

What do u think?

Reply:
I agree, especially with the secrets part later it becomes very vague which part is "settings" and which part is fixed :)